bindings_ffi/src/lib.rs

```diff
@@ -135,146 +135,157 @@ pub struct FfiConversations {
-            start_time_ns: self.start_time_ns,
-            end_time_ns: self.end_time_ns,
-            limit: self.limit,
-        }
-    }
-}
-
 #[uniffi::export(async_runtime = "tokio")]
-impl FfiConversation {
+impl FfiGroup {
     pub async fn send(&self, content_bytes: Vec<u8>) -> Result<(), GenericError> {
-        let conversation = xmtp::conversation::Conversation::new(
+        let group = MlsGroup::new(
             self.inner_client.as_ref(),
-            self.peer_address.clone(),
-        )
-        .map_err(|e| e.to_string())?;
-        conversation
-            .send(content_bytes)
-            .await
-            .map_err(|e| e.to_string())?;
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+
+        group.send_message(content_bytes.as_slice()).await?;
 
         Ok(())
     }
 
-    pub async fn list_messages(
+    pub async fn find_messages(
         &self,
         opts: FfiListMessagesOptions,
     ) -> Result<Vec<FfiMessage>, GenericError> {
-        Conversations::receive(self.inner_client.as_ref()).map_err(|e| e.to_string())?;
-
-        let conversation = xmtp::conversation::Conversation::new(
+        let group = MlsGroup::new(
             self.inner_client.as_ref(),
-            self.peer_address.clone(),
-        )
-        .map_err(|e| e.to_string())?;
-        let options: ListMessagesOptions = opts.to_options();
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+        group.sync().await?;
 
-        let messages: Vec<FfiMessage> = conversation
-            .list_messages(&options)
-            .await
-            .map_err(|e| e.to_string())?
+        let messages: Vec<FfiMessage> = group
+            .find_messages(None, opts.sent_before_ns, opts.sent_after_ns, opts.limit)?
             .into_iter()
             .map(|msg| msg.into())
             .collect();
 
         Ok(messages)
     }
+
+    pub async fn add_members(&self, account_addresses: Vec<String>) -> Result<(), GenericError> {
+        let group = MlsGroup::new(
+            self.inner_client.as_ref(),
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+
+        group.add_members(account_addresses).await?;
+
+        Ok(())
+    }
+
+    pub async fn remove_members(&self, account_addresses: Vec<String>) -> Result<(), GenericError> {
+        let group = MlsGroup::new(
+            self.inner_client.as_ref(),
+            self.group_id.clone(),
+            self.created_at_ns,
+        );
+
+        group.remove_members(account_addresses).await?;
+
+        Ok(())
+    }
 }
 
 #[uniffi::export]
-impl FfiConversation {
-    pub fn id(&self) -> String {
-        self.id.clone()
+impl FfiGroup {
+    pub fn id(&self) -> Vec<u8> {
+        self.group_id.clone()
     }
 }
 
 #[derive(uniffi::Record)]
 pub struct FfiMessage {
-    pub id: String,
+    pub id: Vec<u8>,
     pub sent_at_ns: i64,
-    pub convo_id: String,
+    pub convo_id: Vec<u8>,
```

Comment:
Note for others, we decided on `conversation`/`convo` for the methods in this SDK as opposed to `group`, because the former will be more appropriate when we eventually add 1:1 conversations to libxmtp